app/views/pages/security.html.erb

```diff
@@ -0,0 +1,142 @@
+<% @title = "Security" %>
+
+<div class="l-colspan t-body">
+  <p>
+    Found a security issue with RubyGems or RubyGems.org?
+    Please follow these steps to report it.
+  </p>
+
+  <h2>Reporting a security issue</h2>
+
+  <p>
+    Before continuing, please ensure this is a security issue for the RubyGems
+    client or the RubyGems.org service. For all vulnerabilites with individual
```

Comment:
I like this.  I'd like to better disambiguate for people what the 'RubyGems' client is, but I guess anyone reporting should know, and there's too many words to do it anyhow. Except, maybe a link to the ruby-docs for gem and rubygems.org site respectively?